packages/sdk/src/engine-commands/getDmmf.ts

```diff
@@ -366,7 +367,7 @@ async function getDmmfBinary(options: GetDMMFOptions): Promise<DMMF.Document> {
           },
         ),
         E.map((errorOutputAsJSON: Record<string, string>) => {
-          const defaultMessage = `${chalk.redBright(errorOutputAsJSON.message)}\n`
```

Comment:
> Re: Where did this "logic" go?

This was a leftover newline character from the old `getDmmf` implementation. cc @janpio

Reply:
Ok, that connects those dots.

If you are confident the error message still look similar enough, that's fine.